tableflip/main.go

```
@@ -47,4 +49,15 @@ func main() {
 	<-upg.Exit()
+
+	// Make sure to set a deadline on exiting the process
+	// after upg.Exit() is closed. No new upgrades can be
+	// performed if the parent doesn't exit.
+	time.AfterFunc(30*time.Second, func() {
+		log.Println("Graceful shutdown timed out")
+		os.Exit(1)
+	})
+
+	// Wait for connections to drain.
+	app.ShutdownWithContext(context.Background())
```

golangci-lint (1.62.2)

Error return value of `app.ShutdownWithContext` is not checked

(errcheck)